src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -700,11 +703,22 @@ private static int compareImports(String import1, String import2) {
      *        Line number of current import.
      * @return true, if token have empty line before.
      */
-    private boolean hasEmptyLineBefore(int lineNo) {
+    private int emptyLinesBefore(int lineNo) {
+        int result = 0;
+        final String[] lines = getLines();
         //  [lineNo - 2] is the number of the previous line
         //  because the numbering starts from zero.
-        final String lineBefore = getLine(lineNo - 2);
-        return lineBefore.trim().isEmpty();
+        int lineBeforeIndex = lineNo - 2;
```

Comment:
> Why cannot we use DISTANCE_TO_PREVIOUS_LINE as a constant name?

Problem is that is general distance to previous is "-1" only in context of `lineNo` it become `-2`.
I failed to find good names for that, so I agreed on comment as explanation.